src/Storage/StorageInterface.php

```diff
@@ -20,18 +20,32 @@ interface StorageInterface
      */
     public function all(): array;
 
+    /**
+     * Return feature flag
```

Comment:
Shouldn't you tell that it may throw a RuntimeException ?

Reply:
Sure !